app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -1,9 +1,6 @@
 .gem-c-pagination {
   display: block;
-  margin-top: $gutter;
-  margin-bottom: $gutter;
-  margin-left: -$gutter-half;
-  margin-right: -$gutter-half;
+  margin: govuk-spacing(3);
```

Comment:
This should be `margin: govuk-spacing(6) -(govuk-spacing(3));`

Well, it might be better if it was rebuilt slightly to not use a negative margin, but the consequence of this change is a layout change to the component. Here's the current one:

![Screen Shot 2019-06-10 at 14 20 52](https://user-images.githubusercontent.com/861310/59198180-ff827c00-8b8a-11e9-96ef-6fe0e6c0854c.png)

And here's with this new change:

![Screen Shot 2019-06-10 at 14 20 59](https://user-images.githubusercontent.com/861310/59198191-07dab700-8b8b-11e9-8fea-cbf72986db14.png)

Unless this is intentional?

Reply:
Nope, that was a temp commit with a failed auto-fix on lint which is complaining about `-(govuk-spacing(3))`. It needs to be `margin: govuk-spacing(6) (- govuk-spacing(3) );` to keep both happy (linter and scss) which shows why it was written with separated attributes for each direction 😄